diff --git a/projects/tmux/Dockerfile b/projects/tmux/Dockerfile
index ecbc662e..594f9dd3 100644
--- a/projects/tmux/Dockerfile
+++ b/projects/tmux/Dockerfile
@@ -26,7 +26,7 @@ RUN git clone https://github.com/libevent/libevent.git && \
     make && make install
 
 RUN git clone --depth 1 https://github.com/tmux/tmux.git
-RUN git clone --depth 1 https://github.com/tmux/tmux-fuzzing-corpus.git
+# RUN git clone --depth 1 https://github.com/tmux/tmux-fuzzing-corpus.git
 
 COPY *.c *.dict *.options $SRC/tmux/fuzz/
 COPY *.am $SRC/tmux/
diff --git a/projects/tmux/build.sh b/projects/tmux/build.sh
index 7b0bf3c9..b06f00fe 100644
--- a/projects/tmux/build.sh
+++ b/projects/tmux/build.sh
@@ -34,19 +34,19 @@ MAXLEN=$(grep -Po 'max_len\s+=\s+\K\d+' "${OUT}/input-fuzzer.options")
 
 if [ ! -d "${WORK}/fuzzing_corpus" ]; then
     mkdir "${WORK}/fuzzing_corpus"
-    cd "${WORK}/fuzzing_corpus"
-    bash "${SRC}/tmux/tools/24-bit-color.sh" | \
-        split -a4 -db$MAXLEN - 24-bit-color.out.
-    perl "${SRC}/tmux/tools/256colors.pl" | \
-        split -a4 -db$MAXLEN - 256colors.out.
-    cat "${SRC}/tmux/tools/UTF-8-demo.txt" | \
-        split -a4 -db$MAXLEN - UTF-8-demo.txt.
-    cat "${SRC}/tmux-fuzzing-corpus/alacritty"/* | \
-        split -a4 -db$MAXLEN - alacritty.
-    cat "${SRC}/tmux-fuzzing-corpus/esctest"/* | \
-        split -a4 -db$MAXLEN - esctest.
-    cat "${SRC}/tmux-fuzzing-corpus/iterm2"/* | \
-        split -a5 -db$MAXLEN - iterm2.
-    zip -q -j -r "${OUT}/input-fuzzer_seed_corpus.zip" \
-        "${WORK}/fuzzing_corpus/"
+    # cd "${WORK}/fuzzing_corpus"
+    # bash "${SRC}/tmux/tools/24-bit-color.sh" | \
+    #     split -a4 -db$MAXLEN - 24-bit-color.out.
+    # perl "${SRC}/tmux/tools/256colors.pl" | \
+    #     split -a4 -db$MAXLEN - 256colors.out.
+    # cat "${SRC}/tmux/tools/UTF-8-demo.txt" | \
+    #     split -a4 -db$MAXLEN - UTF-8-demo.txt.
+    # cat "${SRC}/tmux-fuzzing-corpus/alacritty"/* | \
+    #     split -a4 -db$MAXLEN - alacritty.
+    # cat "${SRC}/tmux-fuzzing-corpus/esctest"/* | \
+    #     split -a4 -db$MAXLEN - esctest.
+    # cat "${SRC}/tmux-fuzzing-corpus/iterm2"/* | \
+    #     split -a5 -db$MAXLEN - iterm2.
+    # zip -q -j -r "${OUT}/input-fuzzer_seed_corpus.zip" \
+    #     "${WORK}/fuzzing_corpus/"
 fi
